platform/darwin/src/MGLLight_Private.h

```diff
@@ -0,0 +1,19 @@
+#import <Foundation/Foundation.h>
+
+#import "MGLLight.h"
+
+#import <mbgl/style/light.hpp>
```

Comment:
It may be possible to avoid this import statement too, by forward declaring a class Light inside a namespace style inside a namespace mbgl. See some of the other runtime styling private headers for examples.